test/e2e/self_provisioned_elasticsearch_test.go

```diff
@@ -95,6 +97,17 @@ func (suite *SelfProvisionedTestSuite) TestSelfProvisionedESSmokeTest() {
 	require.NoError(t, err, "Error running smoketest")
 }
 
+func (suite *SelfProvisionedTestSuite) TestValidateEsOperatorImage() {
```

Comment:
Is this being called? I didn't find from where it is invoked.

Reply:
That is the standard test signature when using the [stretchr/testify suite](https://github.com/stretchr/testify) package

Reply:
ah ok I thought that will will not be automatically  invoked.